psalm.xml.dist

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    errorBaseline="psalm-baseline.xml"
+>
+    <projectFiles>
+        <directory name="src"/>
+        <directory name="test"/>
+
+        <ignoreFiles>
+            <directory name="test/_files"/>
+            <directory name="vendor"/>
+        </ignoreFiles>
+    </projectFiles>
+
+    <issueHandlers>
+        <DeprecatedClass>
+            <errorLevel type="suppress">
+                <referencedClass name="Laminas\Di\Resolver\AbstractInjection" />
+                <referencedClass name="LaminasTest\Di\Resolver\AbstractInjectionTest" />
+            </errorLevel>
+        </DeprecatedClass>
+    </issueHandlers>
+
+    <stubs>
+        <file name="./stubs/psr-container.phpstub" />
```

Comment:
I prefer using `lctrs/psalm-psr-container-plugin` but I know that this is a bit opinionated and thus, I know that @Ocramius does not like such magic.

But I'd suggest we go either with the existing psalm-plugin or not.
In both cases, we should drop the stub from this project.

Reply:
And I agree with @Ocramius since all that Plugin does (as far as I got it) is re-implementing what psalm already can do with templates/generics (and therefore adds a greater potential to break or prevent a psalm upgrade).